packages/typescript/chat-core/test/ai/use-ai-context.test.ts

```
@@ -0,0 +1,490 @@
+
+      const size = estimateContextItemSize(items[0]);
+      expect(size).toBeGreaterThan(1000);
+    });
+
+    test('should handle special characters in strings', () => {
+      const items: AIContextItem[] = [
+        {
+          name: 'special-chars',
+          description: 'Item with special characters: 你好, émojis 🎉, and "quotes"',
+          data: {
+            text: 'Special: \n\t\r\b\f\\\/"\'',
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)